Makefile

```diff
@@ -169,7 +169,7 @@ DEFENTROPYSOURCE := /dev/urandom
 
 DEFDISABLEBLOCK := false
 DEFSHAREDFS := virtio-9p
-DEFSHAREDFS_NEMU := virtio-fs
+DEFSHAREDFS_NEMU := virtio-9p
```

Comment:
@chavafg so it seems that this https://github.com/kata-containers/runtime/pull/2054/files#diff-ebce02844da8831ba833219d9a0d0d36R103 will be affected by this change, why we do not use `DEFSHAREDFS` at the configuration.toml just to not have the same variable with the same value

Reply:
I could use `DEFSHAREDFS`, but I think `DEFSHAREDFS_NEMU` should only be used by `configuration-nemu.toml`. nemu related variables will be removed once we deprecate nemu, so I think we should use another variable for `virtio-fs`.
@jodh-intel @egernst any opinion on this?